src/modules/random/index.ts

```diff
@@ -262,15 +262,15 @@ export class RandomModule {
    *
    * @param length The number of digits to generate. Defaults to `1`.
    * @param options The options to use. Defaults to `{}`.
-   * @param options.allowLeadingZeros If true, leading zeros will be allowed. Defaults to `false`.
+   * @param options.allowLeadingZeros If true, leading zeros will be allowed. Defaults to `true`.
```

Comment:
```suggestion
   * @param options.allowLeadingZeros Whether leading zeros are allowed or not. Defaults to `true`.
```

I think we should add the `or not` at the end: https://dictionary.cambridge.org/grammar/british-grammar/whether